features/reports/reports.feature

```diff
@@ -164,17 +167,18 @@ Feature: Correctly formatted reports
       | --no-line-numbers -n             |
 
   Scenario Outline: --single-line shows filename and one line number
-    When I run reek <option> spec/samples/standard_smelly/dirty.rb
+    Given a smelly file called 'smelly.rb'
+    When I run reek <option> smelly.rb
     Then the exit status indicates smells
     And it reports:
       """
-      spec/samples/standard_smelly/dirty.rb -- 6 warnings:
-        spec/samples/standard_smelly/dirty.rb:5: Dirty has the variable name '@s' (UncommunicativeVariableName)
-        spec/samples/standard_smelly/dirty.rb:4: Dirty#a calls @s.title 2 times (DuplicateMethodCall)
-        spec/samples/standard_smelly/dirty.rb:4: Dirty#a calls puts(@s.title) 2 times (DuplicateMethodCall)
-        spec/samples/standard_smelly/dirty.rb:5: Dirty#a contains iterators nested 2 deep (NestedIterators)
-        spec/samples/standard_smelly/dirty.rb:3: Dirty#a has the name 'a' (UncommunicativeMethodName)
-        spec/samples/standard_smelly/dirty.rb:5: Dirty#a has the variable name 'x' (UncommunicativeVariableName)
+      smelly.rb -- 6 warnings:
+        smelly.rb:5: Dirty has the variable name '@s' (UncommunicativeVariableName)
+        smelly.rb:4: Dirty#a calls @s.title 2 times (DuplicateMethodCall)
+        smelly.rb:4: Dirty#a calls puts(@s.title) 2 times (DuplicateMethodCall)
+        smelly.rb:5: Dirty#a contains iterators nested 2 deep (NestedIterators)
+        smelly.rb:3: Dirty#a has the name 'a' (UncommunicativeMethodName)
+        smelly.rb:5: Dirty#a has the variable name 'x' (UncommunicativeVariableName)
```

Comment:
How about  using "minimal_dirty" for examples like  this to cut down distracting output?